user_docs/en/changes.md

```diff
@@ -6,6 +6,12 @@
 
 ### New Features
 
+* Support for math in PDFs has been added.
+This works for formulas with associated MathML mark-up, such as some files generated by newer versions of TeX/LaTeX.
```

Comment:
Isn't "MathML mark-up" somewhat redundant? "Math Mark-up Language mark-up"

```suggestion
This works for formulas with associated MathML, such as some files generated by newer versions of TeX/LaTeX.
```

Reply:
Yes, it is redundant. It is a bit like saying "the attached method is Python code".  Both "code" and "markup" are not needed, but seem to be more common. But I'm too close to MathML, so I defer to others and am fine with your suggestion.